docs/src/apps/rent.md

```diff
@@ -1 +1,45 @@
-# Accounts and Rent
+# Storage Rent for Accounts
+
+Keeping accounts alive incurs storage cost called _rent_ because the cluster must actively maintain the data to process any future transactions on it. This is not like Bitcoin or Ethereum, where account storages don't incur any costs.
+
+The rent is debited from account's balance by the runtime upon the first access in the current epoch by transactions or once per an epoch if there is no transaction. The rent is up-front for the next epoch. If an account can't pay rent for the next epoch, it's purged immediately. The fee is currently a fixed rate, measured in bytes-times-epochs. The fee may change in the future.
+
+Also, there is a way to avoid paying rent with enough account balance called _rent-exempt_.
```

Comment:
"Accounts can be _exempt_ from paying rent by maintaining a minimum balance, this _rent-exemption_ is described below."
Just thought to reframe away from 'avoiding' paying rent.